backends-velox/src/main/scala/org/apache/spark/sql/execution/datasources/velox/VeloxParquetFileFormat.scala

```diff
@@ -62,12 +62,15 @@ class VeloxParquetFileFormat extends GlutenParquetFileFormat
     val parquetOptions = new ParquetOptions(options, sparkSession.sessionState.conf)
     conf.set(ParquetOutputFormat.COMPRESSION, parquetOptions.compressionCodecClassName)
     // pass options to native so that velox can take user-specified conf to write parquet,
-    // i.e., compression and block size.
+    // i.e., compression, block size, block rows.
```

Comment:
Do we need get this config from Spark/Hadoop's configuration? I think it would be better to decided by Spark/Hadoop, instead of Gluten.

Reply:
I agree, will add two configs in gluten config.

Reply:
What I mean is to use pre-defined config from parquet, instead of Gluten defined, cause Gluten should not affect parquet behavior, it should belong to Spark.

Consider this UT from Spark https://github.com/apache/spark/blob/a8ea35f7c2fbd80c3902707ecf224d50800ba622/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala#L1933-L1939

And I could not find parquet.block.rows in parquet-mr project, could you clarify for this config?

Reply:
Parquet or Spark have no config `parquet.block.rows`. This config is used to align with arrow parquet write which has the concept, see `WriterProperties.max_row_group_length`.